bin/ledger2beancount-txns

```diff
@@ -285,7 +285,7 @@ sub print_line($$) {
 # emit fixed beancount header (due to a current beancount limitation, options
 # and plugins must be in the top-level file)
 print "option \"operating_currency\" \"$_\"\n" foreach @operating_currencies;
-print "include \"avg-cost-adjustments.beancount\"\n";
+#print "include \"avg-cost-adjustments.beancount\"\n";
```

Comment:
While i understand that this is not generic and should be dealt with one way or another, simply commenting it out would immediately break my work-flow. So I need a better solution before merging this.